### PR TITLE
Fix PyQt5 qtWebEngineWidgets test

### PR DESCRIPTION
**Pull request by glennra**
Hi Bryan, these are the changes I had to make to get this test to run on my Mac. OSX 10.13.5, Python 2.7.15. 
Ran test using `py.test -k test_PyQt5_QWebEngine_OSX[onedir]`

### CODE REVIEW

File: PyInstaller/hooks/hook-PyQt5.QtWebEngineWidgets.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
                           'QtWebEngineProcess.app']
     # These resource files for QtWebEngineProcess.app must located in .app/Content/Resources
     # see https://doc.qt.io/qt-5.10/qtwebengine-deploying.html
-    datas += [(os.path.join(data_path, *resources, '*'), '.')]
+    datas += [(os.path.join(data_path, *resources), '.')]
     datas += collect_system_data_files(
```
Comment (bjones1) [inline PyInstaller/hooks/hook-PyQt5.QtWebEngineWidgets.py]: I remove these lines, since I think the line above you contributed is probably what's needed.

**Comment by bjones1** [comment]
@glennra, thanks for your help! I've pull this into my branch. Unfortunately, I get failures on Mac OS X with Python 3.6, which I can't debug. What I'm seeing `dyld: Library not loaded: @rpath/QtWebEngineCore.framework/Versions/5/QtWebEngineCore` starting at [line 2210](https://travis-ci.org/bjones1/pyinstaller/jobs/391080269#L2210) if the Travis run. Any ideas? @tallforasmurf, what about you?

**Comment by glennra** [comment]
It looks like the underlying chromium component is failing to start because there is an issue with `test_PyQt5_QWebEngine_OSX.app/Contents/MacOS/content_renderer.service` . Why would this be different for just Python 3.6 and none of the other Python versions?

**Comment by bjones1** [comment]
The Mac tests only run on Python 3.6; no other configurations are tested on Travis. All other tests are on Linux.

Mac is such a pain...

**Comment by glennra** [comment]
Ah, Ok. So it may have something to do with the way that PyQt5-5.10.1-5.10.1-cp35.cp36.cp37.cp38-abi3-macosx_10_6_intel.whl has been built. I am using my own build of Qt5 and PyQt5 (without modifications but omitting the stuff I don't need) also I'm using macports instead of brew, which the test system seems to be using. I'll have a look to see how easy/difficult it will be for me to reproduce this test environment.

**Comment by bjones1** [comment]
Quite possibly. FWIW, the .travis.yml script has the commands used to set up the Mac test environment, which might help.

**Comment by glennra** [comment]
(Note to self) this SO answer gives instructions for running Travis locally https://stackoverflow.com/a/49019950/278403 . I think I will need to set this up to test it. Don't have time right now, but will endeavour to do so at some point in the not-too-distant future.

**Comment by lamourj** [comment]
FYI @glennra I just tested [your branch](https://github.com/glennra/pyinstaller/tree/osx-qt5-qtwebengineprocess-fix) with the latest PyQt5 (5.11.1), and I still run into the `Could not find QtWebEngineProcess` issue.

**Comment by glennra** [comment]
@bjones1 interesting. In the version that works for me I am using builds of Qt5 and PyQt5 that I have built from source. Maybe there is something about the .whl that is different to that. I tried using the same setup as travis but was unable to get it to work yet (ran out of time and gave up).

**Comment by bjones1** [comment]
OK; thanks for trying. Yes, it seems that the wheels and the source do differ in some way (though I'm not sure what exactly the difference is.) I'll close this PR for now; please re-open if you ever figure out how to get it running.

**Pull request closed without merging**